File: src/main/java/kingdoms/client/ClientProxy.java
package kingdoms.client;

import cpw.mods.fml.common.event.FMLPreInitializationEvent;
import kingdoms.server.ServerProxy;
import kingdoms.server.handlers.EntitiesRegister;

public final class ClientProxy extends ServerProxy
{
    @Override
    public void pre(FMLPreInitializationEvent e)
    {
        super.pre(e);
        EntitiesRegister.INSTANCE.render();
        new ClientEvents();
    }
}

File: src/main/java/kingdoms/client/ClientProxy.kt
package kingdoms.client

import cpw.mods.fml.common.event.FMLPreInitializationEvent
import kingdoms.server.ServerProxy
import kingdoms.server.handlers.EntitiesRegister

class ClientProxy: ServerProxy()
{
    override fun pre(e: FMLPreInitializationEvent)
    {
        super.pre(e)
        EntitiesRegister.render()
        ClientEvents()
    }
}